src/Microsoft.Extensions.FileProviders.Null/project.json

```diff
@@ -0,0 +1,34 @@
+﻿{
+  "version": "1.0.0-*",
+  "compilationOptions": {
+    "warningsAsErrors": true,
+    "keyFile": "../../tools/Key.snk",
+    "nowarn": [
+      "CS1591"
+    ],
+    "xmlDoc": true
+  },
+  "description": "Null file and directory providers for Microsoft.Extensions.FileProviders.",
```

Comment:
That won't work if we want to reuse the types from `Microsoft.Extensions.FileProviders.Sources`. To avoid the circular dependency, we would either move those types to abstractions or move the null provider to sources both of which feels like incorrect layering.